concepts/objects/about.md

```diff
@@ -0,0 +1,251 @@
+# About
+
+## Explanation
+
+Besides primitive data types like `number` and `string`, there is another important data type in JavaScript called `object`.
+Objects are collections of key-value pairs.
+In other languages they are referred to as maps or dictionaries and often times values need to have the same data type.
+In JavaScript, only the type of the key is restricted, it has to be a string.
+The values inside one object can have different types.
+They can be primitive types like numbers but also arrays, other objects or even functions.
+
+## Creating an Object
+
+You create an object (literal) with curly brackets.
+You can also directly include some entries (key-value pairs).
+For that, state the key first, followed by a colon and the value.
+
+```javascript
+const emptyObject = {};
+
+const obj = {
+  favoriteNumber: 42,
+  greeting: 'Hello World',
+  useGreeting: true,
+  address: {
+    street: 'Trincomalee Highway',
+    city: 'Batticaloa',
+  },
+  fruits: ['melon', 'papaya'],
+  addNumbers: function (a, b) {
+    return a + b;
+  },
+};
+```
+
+The trailing comma after the last entry is optional in JavaScript.
```

Comment:
It may be a good idea to mention why one would want to add a trailing comma (better diffs etc).

Reply:
Provided a link where this is spelled out. I think that is sufficient.